src/docs/Contributing/Designers/AddNewIcons/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Add new icons'
+order: 4
+---
+
+> First of all, each icon must be reviewed by the designer(s) in charge of the quality and the consistency of the icons at LEROY MERLIN. Second of all, each new icon must be declined in the following sizes: `24px`, `32px`, `48px` & `64px`. Here are the steps that you must follow to add new icons in Mozaic.
+
+## 1. Get in touch with Mozaic's team
+
+It's important to discuss your needs and what can be produced together before starting to create the said icon. Our icon designer is [Simon Roger](mailto:simon.roger@ext.leroymerlin.fr) for now. Get in touch with him to speak about format and sizes of your icon(s).
```

Comment:
Maybe we can also mention the fact that they can contact the team on Slack, or create a Github "Issue" using the templates: https://github.com/adeo/mozaic-design-system/issues/new/choose

What do you think?

Reply:
yes having more than one entry point is better. I'll add this